tests/golos.publication_tests.cpp

```diff
@@ -50,7 +50,7 @@ class golos_publication_tester : public golos_tester {
 
         produce_block();
         create_accounts(_users);
-        create_accounts({cfg::charge_name, cfg::token_name, cfg::vesting_name, cfg::emission_name, N(dan.larimer)});
+        create_accounts({cfg::charge_name, cfg::token_name, cfg::vesting_name, cfg::emission_name, cfg::control_name, N(notify.acc), N(dan.larimer)});
```

Comment:
Why do you need cfg::control_name & N(notify.acc) accounts?

Reply:
cfg::control_name is needed as it is used in the create_vesting method
https://github.com/GolosChain/golos.contracts/blob/1e48eeb6c4ea8044c94441a8e50dbcd6505f18b9/tests/golos.vesting_test_api.hpp#L15